lib/active_merchant/billing/gateways/orbital.rb

```diff
@@ -565,15 +565,15 @@ def get_address(options)
       end
 
       def add_safetech_token_data(xml, payment_source, options)
-        xml.tag! :CardBrand, options[:card_brand]
-        xml.tag! :AccountNum, payment_source
+        xml.tag! :CardBrand, payment_source.split(';')[1]
```

Comment:
💬 Style:

maybe is just matter of taste on semantics => `payment_source.split(';').first`

Reply:
you 're right updated